core/bignum/bit_or_spec.rb

```diff
@@ -24,9 +24,11 @@
   end
 
   it "raises a TypeError when passed a Float" do
-    lambda {
-      bignum_value | bignum_value(0xffff).to_f
-    }.should raise_error(TypeError)
+    not_supported_on :opal do
+      lambda {
+        bignum_value | bignum_value(0xffff).to_f
```

Comment:
Or is it because that value is just another Numeric and it's not possible to differentiate it in Opal?
And the spec below should pass?